Assets/Mirror/Editor/Weaver/Processors/TargetRpcProcessor.cs

```diff
@@ -125,10 +125,7 @@ public static MethodDefinition ProcessTargetRpcCall(TypeDefinition td, MethodDef
 
         public static bool ProcessMethodsValidateTargetRpc(TypeDefinition td, MethodDefinition md, CustomAttribute ca)
         {
-            const string targetPrefix = "Target";
-            int prefixLen = targetPrefix.Length;
-
-            if (md.Name.Length > prefixLen && md.Name.Substring(0, prefixLen) != targetPrefix)
```

Comment:
This actually had 2x bug because someone wanted to be smart and let the compiler find the length of the string. To have been consistent with the original bugged code, it should have compared against 5.